src/components/map/MapService.js

```diff
@@ -1667,14 +1671,19 @@ goog.require('ga_urlutils_service');
           if (!extent || extent.length !== 4) {
             return gaGlobalOptions.defaultExtent;
           }
-          return [
+          var extent = [
             Math.max(extent[0], gaGlobalOptions.defaultExtent[0]),
             Math.max(extent[1], gaGlobalOptions.defaultExtent[1]),
             Math.min(extent[2], gaGlobalOptions.defaultExtent[2]),
             Math.min(extent[3], gaGlobalOptions.defaultExtent[3])
           ];
+          if (!isExtentEmpty(extent)) {
+            return extent;
+          }
```

Comment:
if there is no intersection with the defaultExtent boundaries I think that makes sense that we return `undefined`.